wallet/src/main/scala/org/bitcoins/wallet/DLCWallet.scala

```diff
@@ -362,53 +371,30 @@ abstract class DLCWallet extends Wallet {
   }
 
   def verifyCETSigs(sign: DLCSign): Future[Boolean] = {
```

Comment:
Now that we have a dedicated verifier, these functions should probably be moved to there